gse-app/src/main/java/com/powsybl/gse/app/ProjectPane.java

```diff
@@ -203,6 +216,119 @@ protected void updateItem(Object value, boolean empty) {
                     setGraphic(null);
                 } else {
                     fillCellInfosForObject(value);
+                    if (value == null) {
```

Comment:
General comment pour updateItem. This method is too big. You should decompose into several methods.